README.md

```diff
@@ -1,2 +1,3 @@
 # WikiPreview.Fluent.Plugin
+
 Search in Wikipedia and displays results directly in the Fluent Search.
```

Comment:
Add descriptions how to use the plugin with a screenshot, look at the other repos.